build/Dockerfile

```diff
@@ -5,7 +5,7 @@ ENV OPERATOR=/usr/local/bin/openshift-pipelines-operator \
     USER_NAME=openshift-pipelines-operator
 
 # install operator binary
-COPY build/_output/bin/openshift-pipelines-operator ${OPERATOR}
+COPY build/_output/bin/tektoncd-pipeline-operator ${OPERATOR}
```

Comment:
i understand why this change is required. But at present this will upset our ci systems and other operator-sdk commands which expect the repository root name to be same as the operator name.

As a workaround, you can check out this repository as `openshift-pipelines-operaotr`
```
git clone git@github.com:openshift/tektoncd-pipeline-operator.git openshift-pipelines-operator
```

or you can create a symbolic link with the name `openshift-pipelines-operator` to tektoncd-pipeline-operator directory

could you remove this change from this patch.

Reply:
Agree with @nikhil-thomas  !!!